experiments/AMIP/moist_mpi_earth/coupler_utils/calendar_timer.jl

```diff
@@ -3,29 +3,35 @@
 
 """
     current_date(t)
+
 Return the model date
 """
 current_date(t) = date0 + Dates.Second(t)
 
 """
     strdate_to_datetime(strdate)
+
 Convert from String ("YYYYMMDD") to Date format  
 """
 strdate_to_datetime(strdate::String) =
-    Dates.Date(parse(Int, strdate[1:4]), parse(Int, strdate[5:6]), parse(Int, strdate[7:8])) # required by the official AMIP input files
+    Dates.DateTime(parse(Int, strdate[1:4]), parse(Int, strdate[5:6]), parse(Int, strdate[7:8])) # required by the official AMIP input files
```

Comment:
is this hardcoded to a specific format? should we concerned that that could change?

Reply:
If we use `Dates`, then `Date()` and `DateTime()` are the default formats, so I would have thought that this is quite safe.